Integrating with your webapp/README.md

```diff
@@ -0,0 +1,167 @@
+
+
+## 2. Building a simple web app to stream your Android applications
+
+Create a simple webpage containing an element with the ID `anbox-stream`. This is where the video stream will be displayed:
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+    <meta content="utf-8" http-equiv="encoding">
+    <title>Anbox Streaming SDK Example</title>
+</head>
+<body>
+    <div id="anbox-stream"></div>
+</body>
+```
+
+Then include the stream SDK:
+
+```sh
+$ git clone https://github.com/anbox-cloud/anbox-streaming-sdk.git
+```
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+    <meta content="utf-8" http-equiv="encoding">
+    <title>Anbox Cloud Tutorial</title>
+    <script src="./anbox-streaming-sdk/js/anbox-stream-sdk.js"></script>
+</head>
+<body>
+    <div id="video-stream"></div>
+</body>
+```
+
+This will inject two classes in the `window` object.
+ * `window.AnboxStream` is the main object you'll interact with. It handles the actual WebRTC communication
+ * `window.AnboxStreamGatewayConnector` is a middleware that handles authentication for the Stream Gateway. If you want to later add your own authentication and application layer, you can implement your own connector without having to change the rest of the SDK
+
+----
+
+Streaming works as follows:
+ 1. Authenticate with the Stream Gateway
+ 2. Pick an application and configure the stream
+ 3. Let the `AnboxStream` class handle the WebRTC setup
+
+#### 1. Authenticate with the Stream Gateway
+
+Authentication is done through a **connector**. This is a small class that is used later on by the AnboxStream object
+and whose purpose is to handle authentication. A connector for the Stream Gateway is provided, but you must write
+your own to use a middleware (this is covered in the next step of the tutorial).
+
+The Stream Gateway has a token based authentication. You can obtain a new token with the following command:
+```sh
+# Anbox Cloud Appliance based
+$ anbox-cloud-appliance gateway account create my-account-name
+
+# Juju based
+$ juju run-action anbox-stream-gateway/0 create-auth-token service=my-account-name --wait
+```
+
+Next, create the connector:
+```js
+const connector = new AnboxStreamGatewayConnector({
+    url : 'gateway.url.net',
+    authToken: 'YOUR_AUTH_TOKEN',
+    session: {
+        app: "my-android10-app",
+    },
+});
+```
+
+> If you've deployed Anbox Cloud through the AWS marketplace image, you can replace `gateway.url.net` by `https://<ec2_ip>`.
+> On Juju based deployments, you would use the IP address of `anbox-stream-gateway-lb` has such: `https://<lb_ip>`
+
+----
+
+Create a new `AnboxStream` object:
+```js
+let stream = new AnboxStream({
+    connector: connector,
+    targetElement: "anbox-stream",
+    screen: {
```

Comment:
As discussed the `screen` has to be moved out from AnboxStream to AnboxStreamGatewayConnector object, we pass the screen info of a container when interacting with stream gateway.